<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KanjiVocabDialog</class>
 <widget class="QDialog" name="KanjiVocabDialog">
  <property name="windowTitle">
   <string>Tag vocabulary cards based on kanji</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="kanjiGroupBox">
     <property name="title">
      <string>Kanji cards</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="kanjiFieldLayout">
        <item>
         <widget class="QLabel" name="kanjiFieldLabel">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Field:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="kanjiFieldCombo"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="kanjiFilterLayout">
        <item>
         <widget class="QLabel" name="kanjiFilterLabel">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Filter pattern:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="kanjiFilterLineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="vocabGroupBox">
     <property name="title">
      <string>Vocabulary cards</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="vocabFieldLayout">
        <item>
         <widget class="QLabel" name="vocabFieldLabel">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Field:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="vocabFieldCombo"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="includeNonKanjiCheckBox">
        <property name="text">
         <string>Include cards not containing any kanji</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="tagsGroupBox">
     <property name="title">
      <string>Tags</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="tagsLayout">
        <item>
         <widget class="QLabel" name="tagsLabel">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Tags to add:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="tagsLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="deleteTagsCheckBox">
        <property name="text">
         <string>Delete these tags from non-matching cards</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>KanjiVocabDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>KanjiVocabDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
